src/database/seeds/5-create-shopping-lists.seed.ts

```diff
@@ -13,12 +13,35 @@ export default class CreateShoppingLists implements Seeder {
         {
           name: 'Churrasco',
           user: {
-            id: 'user',
+            id: 'user1',
           },
           products: [
             {
               id: 1,
             },
+            {
+              id: 7,
+            },
+          ],
+        },
+        {
+          name: 'Mes',
```

Comment:
Don't be afraid to use special characters :D.

```suggestion
          name: 'Mês',
```